man/fi_mr.3.md

```diff
@@ -740,6 +775,11 @@ The follow flag may be specified to any memory registration call.
   API AllocHost call (as opposed to using malloc-like host allocation,
   unified/shared memory allocation, or AllocDevice).
 
+*FI_MR_DMABUF*
```

Comment:
Should this also be a capability bit so that the application can check if this supported by the provider?

Reply:
Maybe, and I had the same thought.  Can the current FI_HMEM work, which would require providers support FI_HMEM to also support dmabuf registration?  I'd like to avoid capability bit overload when possible.  But I'm not sure if FI_HMEM is sufficient.

I was thinking that a provider could call mmap() to map the region into the VA space, then treat the dmabuf the same as a VA registered region.  The provider may need to discover what hmem API was used to allocate the region.  Is there an issue if the provider were forced down this path?  IOW, dmabuf support becomes an optimization that the provider could leverage, otherwise it needs to fallback to whatever its standard HMEM support is.